<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="concurrency.stage2.AppTest" tests="1" skipped="0" failures="0" errors="0" timestamp="2022-09-14T13:29:19" hostname="cho-choui-MacBookAir.local" time="1.499">
  <properties/>
  <testcase name="test" classname="concurrency.stage2.AppTest" time="1.499"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[Exception in thread "Thread-5" java.lang.RuntimeException: java.net.http.HttpTimeoutException: request timed out
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:26)
	at concurrency.stage2.AppTest.lambda$test$0(AppTest.java:113)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.http.HttpTimeoutException: request timed out
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:559)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:119)
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:24)
	... 2 more
Exception in thread "Thread-6" java.lang.RuntimeException: java.net.http.HttpTimeoutException: request timed out
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:26)
	at concurrency.stage2.AppTest.lambda$test$0(AppTest.java:113)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.http.HttpTimeoutException: request timed out
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:559)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:119)
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:24)
	... 2 more
Exception in thread "Thread-7" java.lang.RuntimeException: java.net.http.HttpTimeoutException: request timed out
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:26)
	at concurrency.stage2.AppTest.lambda$test$0(AppTest.java:113)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.http.HttpTimeoutException: request timed out
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:559)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:119)
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:24)
	... 2 more
Exception in thread "Thread-8" java.lang.RuntimeException: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:26)
	at concurrency.stage2.AppTest.lambda$test$0(AppTest.java:113)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:555)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:119)
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:24)
	... 2 more
Caused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:68)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.purgeTimeoutsAndReturnNextDeadline(HttpClientImpl.java:1248)
	at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:877)
Caused by: java.net.ConnectException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:69)
	... 2 more
Exception in thread "Thread-9" java.lang.RuntimeException: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:26)
	at concurrency.stage2.AppTest.lambda$test$0(AppTest.java:113)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:555)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:119)
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:24)
	... 2 more
Caused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:68)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.purgeTimeoutsAndReturnNextDeadline(HttpClientImpl.java:1248)
	at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:877)
Caused by: java.net.ConnectException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:69)
	... 2 more
Exception in thread "Thread-10" java.lang.RuntimeException: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:26)
	at concurrency.stage2.AppTest.lambda$test$0(AppTest.java:113)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:555)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:119)
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:24)
	... 2 more
Caused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:68)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.purgeTimeoutsAndReturnNextDeadline(HttpClientImpl.java:1248)
	at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:877)
Caused by: java.net.ConnectException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:69)
	... 2 more
Exception in thread "Thread-11" java.lang.RuntimeException: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:26)
	at concurrency.stage2.AppTest.lambda$test$0(AppTest.java:113)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:555)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:119)
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:24)
	... 2 more
Caused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:68)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.purgeTimeoutsAndReturnNextDeadline(HttpClientImpl.java:1248)
	at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:877)
Caused by: java.net.ConnectException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:69)
	... 2 more
Exception in thread "Thread-12" java.lang.RuntimeException: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:26)
	at concurrency.stage2.AppTest.lambda$test$0(AppTest.java:113)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:555)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:119)
	at concurrency.stage2.TestHttpUtils.send(TestHttpUtils.java:24)
	... 2 more
Caused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:68)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.purgeTimeoutsAndReturnNextDeadline(HttpClientImpl.java:1248)
	at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:877)
Caused by: java.net.ConnectException: HTTP connect timed out
	at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:69)
	... 2 more
]]></system-err>
</testsuite>
